chrome-launcher/test/chrome-launcher-test.ts

```diff
@@ -6,59 +6,97 @@
 'use strict';
 
 import {Launcher} from '../chrome-launcher';
-
+import {spy, stub} from 'sinon';
 import * as assert from 'assert';
 
 const log = require('lighthouse-logger');
 const fsMock = {
   openSync: () => {},
-  closeSync: () => {}
+  closeSync: () => {},
+  writeFileSync: () => {}
 };
 
 describe('Launcher', () => {
-  it('accepts and uses a custom path', async () => {
+  beforeEach(() => {
     log.setLevel('error');
-    let callCount = 0;
-    const rimrafMock = (_: string, done: () => void) => {
-      callCount++;
-      done();
-    };
+  });
+
+  afterEach(() => {
+    log.setLevel('');
+  });
+
+  it('sets no default flags', async () => {
+    const spawnStub = stub().returns({pid: 'some_pid'});
+
+    const chromeInstance = new Launcher(
+        {userDataDir: 'some_path', noDefaultFlags: true},
+        {fs: fsMock as any, rimraf: spy() as any, spawn: spawnStub as any});
+    stub(chromeInstance, 'waitUntilReady').returns(Promise.resolve());
+
+    chromeInstance.prepare();
+
+    try {
+      await chromeInstance.launch();
+    } catch (err) {
+      return Promise.reject(err);
+    }
+
+    const chromeFlags = spawnStub.getCall(0).args[1] as string[];
```

Comment:
s/chromeFlags/launchedFlags

to help differentiate with the `this.chromeFlags` of the launcher